build.gradle

```diff
@@ -97,4 +99,9 @@ idea {
     languageLevel = '1.11'
     vcs = 'Git'
   }
+  module {
+      def generatedZooKeeper = Paths.get(project(':persistence-zookeeper').protobuf.generatedFilesBaseDir, "main", "java")
+      sourceDirs += file(generatedZooKeeper)
+      generatedSourceDirs+= file(generatedZooKeeper)
```

Comment:
IntelliJ code completion does not work even though all docs say this is the way to do it. I'm happy for any help here.